components/buildless-serverless/internal/controller/resources/deployment_test.go

```diff
@@ -83,6 +83,9 @@ func TestDeployment_construct(t *testing.T) {
 		expectedLabels := map[string]string{
 			"app": "test-function-name",
 			"serverless.kyma-project.io/function-name": "test-function-name",
+			"serverless.kyma-project.io/managed-by":    "buildless-function-controller",
+			"serverless.kyma-project.io/resource":      "deployment",
+			"serverless.kyma-project.io/uuid":          "",
```

Comment:
you can test this line by adding the constant uid to the function returned by the `minimalFunction` func. I would suggest to test this case

Reply:
Thanks for that